src/utils/rest-api.js

```diff
@@ -1887,7 +1887,7 @@ export function createSubstation(
         substationCountry: substationCountry === '' ? null : substationCountry,
         properties: asObj,
     });
-    console.debug('createSubstation body', properties, body);
+    console.debug({ body });
```

Comment:
a little message can be nice to understand better what we see in the console

Reply:
There is `console.info('Creating substation creation');` some lines above, so I thought it could suffice.

Reply:
I was talking about the fact that we don't know that it's the body, see the previous message that you changed it was better

Reply:
I was talking about the fact that we don't know that it's the body, see the previous message that you changed it was better